CondFormats/EcalObjects/interface/EcalCATIAGainRatios.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _CondFormats_EcalObjects_EcalGainRatios_h_
+#define _CondFormats_EcalObjects_EcalGainRatios_h_
```

Comment:
Include guards should have the format
PackageName_SubPackageName_FileName_h

Rule 4.1
http://cms-sw.github.io/cms_coding_rules.html